src/test/php/net/xp_framework/unittest/DemoTest.class.php

```diff
@@ -1,94 +1,87 @@
-      $this->fail('Ignored test executed', 'executed', 'ignored');
-    }
+  /**
+   * A test that is ignored
+   *
+   */
+  #[@test, @ignore('Ignored')]
+  public function ignored() {
+    $this->fail('Ignored test executed', 'executed', 'ignored');
+  }
 
-    /**
-     * Setup method
-     *
-     */
-    public function setUp() {
-      if (0 == strcasecmp('alwaysSkipped', $this->name)) {
-        throw new PrerequisitesNotMetError('Skipping', null, $this->name);
-      }
+  /**
+   * Setup method
+   *
+   */
+  public function setUp() {
+    if ('alwaysSkipped' === $this->name) {
+      throw new PrerequisitesNotMetError('Skipping', null, $this->name);
     }
+  }
 
-    /**
-     * A test that is skipped due to not met prerequisites.
-     *
-     */
-    #[@test]
-    public function alwaysSkipped() {
-      $this->fail('Skipped test executed', 'executed', 'skipped');
-    }
+  /**
+   * A test that is skipped due to not met prerequisites.
+   *
+   */
+  #[@test]
+  public function alwaysSkipped() {
+    $this->fail('Skipped test executed', 'executed', 'skipped');
+  }
 
-    /**
-     * A test that always fails because of a failed assertion
-     *
-     */
-    #[@test]
-    public function alwaysFails() {
-      $this->assertTrue(FALSE);
-    }
+  /**
+   * A test that always fails because of a failed assertion
+   *
+   */
+  #[@test]
+  public function alwaysFails() {
+    $this->assertTrue(false);
+  }
 
-    /**
-     * A test that always fails because the expected exception was not
-     * thrown.
-     *
-     */
-    #[@test, @expect('lang.IllegalArgumentException')]
-    public function expectedExceptionNotThrown() {
-      TRUE;
-    }
+  /**
+   * A test that always fails because the expected exception was not
+   * thrown.
+   *
+   */
+  #[@test, @expect('lang.IllegalArgumentException')]
+  public function expectedExceptionNotThrown() {
+    true;
+  }
 
-    /**
-     * A test that always fails because an exception is thrown.
-     *
-     */
-    #[@test]
-    public function throwsAnException() {
-      throw new IllegalArgumentException('');
-    }
+  /**
+   * A test that always fails because an exception is thrown.
+   *
+   */
+  #[@test]
+  public function throwsAnException() {
+    throw new IllegalArgumentException('');
+  }
 
-    /**
-     * A test that timeouts
-     *
-     */
-    #[@test, @limit(time= 0.1)]
-    public function timeouts() {
-      $start= gettimeofday();
-      $end= (1000000 * $start['sec']) + $start['usec'] + 1000 * 200;    // 0.2 seconds
-      do {
-        $now= gettimeofday();
-      } while ((1000000 * $now['sec']) + $now['usec'] < $end);
-    }
+  /**
+   * A test that timeouts
+   *
+   */
+  #[@test, @limit(time= 0.1)]
+  public function timeouts() {
+    $start= gettimeofday();
+    $end= (1000000 * $start['sec']) + $start['usec'] + 1000 * 200;    // 0.2 seconds
+    do {
+      $now= gettimeofday();
+    } while ((1000000 * $now['sec']) + $now['usec'] < $end);
   }
```

Comment:
IIRC this was done because usleep() didn't work reliably on some platforms. Guess this can go by now.